package.json

```diff
@@ -17,7 +17,7 @@
     "test": "jest --runInBand --forceExit"
   },
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
```

Comment:
:thinking: this looks like a (small) breaking change for people using the package directly. But it's probably OK to do such BC breaks for now.